scanapp/tasks.py

```diff
@@ -188,3 +229,17 @@ def fill_unfilled_scan_model(scan, files_count, scancode_notice, scancode_versio
     scan.scancode_version = scancode_version
     scan.save()
     return scan
+
+
+def parse_url(URL):
+    """
+        Parses the URL and checks if it's a git URL. If it is a git URL then the flag is set to 1.
+    """
+    flag = 0
+    g = GitURL(URL)
+
+    try:
```

Comment:
The function which I am using to check whether it is a git url or not would return null in the case of a random absurd url but we to return the 'flag' variable every time. So the try block.